### DISCUSSION TITLE
Rune Audio site down for good ?

### DISCUSSION THREAD

**Discussion by NexusPenguin** [rern/rAudio#1279] [General]
Hi,

Since 2 days I am trying to access the Rune audio website. 
I thought maybe I could find a previous version of Rune Audio for Pi4 as the current one does not display lyrics anymore it seems :-(

But I get a "NET::ERR_CERT_DATE_INVALID" which is obviously an outdated certificate. 

So is the site down for good or will it come back online at some point ? 

Thanks a lot for your answers. 

Regards.

**Comment by klassikmann** [comment]
@NexusPenguin 
You are here on the site of rAudio, the successor of both "RuneAudio" and "RuneAudio+R e6" with 100 releases the last two years. Firefox  prohibits access to RuneAudio, presumably because of an expired TLS certificate **and** the requirement of HSTS . Using Chrome or Edge permit access after defining an exception.

**Comment by NexusPenguin** [comment]
Hi,

Yes, thanks, found the "Releases" and all the successive image files. 

So I could get an older one, that I know FOR SURE was able to display almost ANY lyrics.

Now, whatever version I use : NO LYRICS. Never...

There seems to be a coincidence between "RuneAudio" site down, and "Lyrics Not available". 

So I am wondering if those could be related... If not, is there an explanation as to why I am suddenly not getting ANY lyrics anymore ?

**Reply by klassikmann** [reply]
@NexusPenguin 
> So I could get an older one, that I know FOR SURE was able to display almost ANY lyrics.

My advice: Explore the latest release and look at the new features. If anything is wrong with lyrics, rern will solve the problem.
btw. I have no problems with lyrics (existing and new ones). 
@rern But I see a permission issue: Existing lyrics have permissions 0755 http:http, new ones 0644 root:root


> There seems to be a coincidence between "RuneAudio" site down, and "Lyrics Not available".

I definitely don't see any connection.

**Comment by NexusPenguin** [comment]
Hi klassikmann,

Thanks a lot for your answers. 

Well, given the November and december releases did not help solving the issue, I switched back to latest image, but haven't upgraded to 20231003 so far. 

The problems stays the same : if I hit the title of a song (i.e. Loreena McKENNITT - Caravanserai") I get a result saying "Loreena McKennit" and then a black block saying "Lyrics not available". 

I am probably doing something wrong with a setting somewhere... But I wonder what it can be :-(

I'm a taker of ANY idea :-)

Regards. 

FRED

**Reply by ceejayemm** [reply]
@NexusPenguin 
I have no problem with the latest version on a RPi 3B+ 

Is the Features / Lyrics in File option enabled or disabled ?  Mine is DISABLED.

Chris

**Reply by NexusPenguin** [reply]
Hi Chris,

Enabled or Disabled : problem still the same here :-(

Is there a way to retrieve a log to see why I do not get an answer when I ask for Lyrics ? 

Just to make things sure : I have reformatted the SD card completely before flashing rAudio. 
I'm waiting for that complete erasure to finish and then I'll flash. 

I assume the lyrics are retrieved online. 
Maybe I should enable a port on my router ? 
But that would be weird as I have not touched anything since it was working (on the router).

**Reply by ceejayemm** [reply]
@NexusPenguin 
Mine just 'works out of the box', no changes to rAudio-1 or my router. I think the lyrics are fetched from the internet on the fly, whether there is a log you can look at I don't know - sorry. Is there any firewall between your RPi and the internet that might be blocking such access ?

Got to the limit of my knowledge on this subject now :-)

Chris

**Reply by NexusPenguin** [reply]
Hi Chris,

Thanks a lot. 

Same here on both points : 
- it used to retrieve lyrics out of the box until a couple of days ; 
- I am more or less at the limit of my knowledge too :-)

Lyrics where the main reason for my choice of RuneAudio. So I hope I will be able to get them back anyways soon :-)

But since I can display them on the "remote" device using Symfonium on Android, I can more or less compensate with those. 
Not as flexible because they have to be included in metadata or.lrc file, but still... Kind of a workaround. 

Regards. 

FRED

**Comment by NexusPenguin** [comment]
Hi, 

Replying to myself because the problem is solved. 

As a lot of people reported taht their "Lyrics" were showing as they should, I wondered why mine did not. 

It was commonly said that Lyrics are retrieved over internet on the fly. 
So this of course supposes that you have an active internet connection. 

I thought that having the RaspBerry Pi4 connected to an Ethernet switch that is itself connected to an internet connected router was enough. 

But in doubt I turned on Wi-Fi and connected it to my local Wi-Fi network. 
And... Voilà. Lyrics again. 

The funny thing is that I got "Bio" of artist. So I assumed that I was connected. 
But it seems not :-(

Anyways : with Wi-Fi turned on, Lyrics are there again... Would be nice to allow even if Wi-Fi is turned off.

**Reply by cmh714** [reply]
I dont use wifi at all. It works, you have something messed up in your network. GL!

**Reply by NexusPenguin** [reply]
Hi,

Thanks a lot for that info. 

After your message I turned off Wi-Fi again. 

And : guess what ? 

Lyrics are still there it seems :-)

Funny thing is it not ? The ways of computer world are sometimes difficult to understand...

Regards.

**Reply by cmh714** [reply]
Nuts!! When in doubt reboot several times ;^) Glad youre working mostly

**Reply by NexusPenguin** [reply]
Hi,

I did reboot several times. 

I even installed rAudio on a freshly formatted SD Card to mbe sure. 

No lyrics. But I had internet for sure as I could update to latest version of rAudio AND could retrieve Artist bio. 

So turning on Wi-Fi and turning it off helped it seems. 
Whatever the mysteries that lay beyond any comprehension of mine, it works and that is all I care about :-)

Still, dev may would like to give that a look if he hs a change and the time :-)

Regards.

**Comment by ceejayemm** [comment]
For info

The runeaudio,com site seems to be back up again now. Somebody obviously fixed the certificate issue.

Chris

**Reply by NexusPenguin** [reply]
Hi,

Thanks for the info. 

Can it be that the fact that it came back online is related to Lyrics that came back ? 
Is it possible that retrieving the Lyrics goes through the rUne Audio webstie somehow ? 

Anyways, as I said, I got Lyrics back on rAudio. 
Would now be happy to have metter Album management in Library. 

Regards

**Reply by ceejayemm** [reply]
@NexusPenguin 

I would hardly think so, why would an independent application be reliant on an unrelated web site ?  I know rAudio-1 was spun off from RuneAudio but I am pretty sure Rern would NOT have retained any such dependency.

**Reply by NexusPenguin** [reply]
Hi,

OK. Just a coincidence then. Was just wondering... But as you do I think the two things are most probably not related
